pkg/api/adjustment.go

```diff
@@ -378,3 +396,9 @@ func (a *ContainerAdjustment) initLinuxResourcesUnified() {
 		a.Linux.Resources.Unified = make(map[string]string)
 	}
 }
+func (a *ContainerAdjustment) initLinuxRdt() {
```

Comment:
Just a note: #118 has a bunch of queued similar changes, updating the result-handling code so that there is no need to always a priory create a fully constructed but empty adjustment. It would be nice to get that finally in, so I'll rebase it that on the latest main/HEAD then update with any missing init*() bits necessary.

Reply:
:+1: I'll review when that's updated. And then rebase when that's merged